### DISCUSSION TITLE
how to use "serial adapter uploader" with platformIO ?

### DISCUSSION THREAD

**Discussion by cpainchaud** [SpenceKonde/megaTinyCore#408] [Q&A] [answered]
Hi there!

First of all, thank you for all the hard work! I got this running on Arduino 1.8.13 and the Blinking LED example using the "Serial adapter + 4.7k (pyudpi style)
![image](https://user-images.githubusercontent.com/6696638/117039371-fa8fbe00-ad08-11eb-90d7-979ac761b46c.png)

I am trying to find this option name in https://github.com/SpenceKonde/megaTinyCore/blob/master/PlatformIO.md but I have had no luck so far .

Thank you in advance !

**Comment by Modac** [comment, marked as answer]
I've never tried it myself but did you see this section?
https://github.com/SpenceKonde/megaTinyCore/blob/master/PlatformIO.md#pyupdi

**Reply by cpainchaud** [reply]
I found it like a few seconds ago after 2 hours of trial .... it's always like that: you don't find it, you ask a question on internet and you find the answer after 10 minutes!
So I tried and it just worked!

 Thank for answering!
